queries/dune_v2/period_slippage.sql

```diff
@@ -384,17 +381,14 @@ final_token_balance_sheet as (
         tx_hash,
         block_time
     having
-        sum(amount) != 0
-),
-
--- Benchmark: 4 minutes for 1 day (non-pro account)
--- select * from final_token_balance_sheet limit 10
-token_times as (
+        sum(amount) != cast(0 as int256)
```

Comment:
I know this is not your fault, but Dune's. But, I can't fathom why we have to do this, this feels very stupid.

Reply:
My bad, I was just venting at the fact that when doing `sum(amount) != 0` we have to cast the `0` to `int256`. I'd assume any programming language smart enough to know that.